developer-guide.md

```diff
@@ -169,3 +179,34 @@ The [developer console](https://developers.chrome.com/origintrials/#/trials/acti
   - User Subset: A small percentage of Chrome users will always be excluded from the trial, even
       when a valid token is provided. The exclusion percentage varies for each trial, but is
       typically less than 5%.
+  ### 21. What does the _Valid Until_ date mean for my tokens?
+  - Tokens are guaranteed to be accepted by Chrome up to the _Valid Until_ date. It is calculated
+      based on your feedback, initially set to 6 weeks after the token is created.
+  - Feedback is required every 6 weeks to extend the _Valid Until_ date in order to keep
+      participating in the trial. 
+  - Before the _Valid Until_ date arrives, we'll send an email to invite you to provide feedback
+      and continue participating in the trial.
+  - Tokens without any feedback in the past 6 weeks will automatically be disabled via remote
+      processes after a grace period.
+  - We will send a last warning email before disabling the token. This will be your last chance
+      to provide feedback and keep the token active. After that your token will stop working on
```

Comment:
Maybe use "accepted" as earlier, and be more explicit about what "stop working" means?
```suggestion
      to provide feedback and keep the token active. After that your token will no longer be accepted by
```